Specs/Scene/ExpressionSpec.js

```diff
@@ -524,4 +534,12 @@ defineSuite([
         expression = new Expression(new MockStyleEngine(), 'isFinite(Color("white"))');
         expect(expression.evaluate(undefined)).toEqual(false);
     });
+
+    it('evaluates ternary conditional', function() {
```

Comment:
Add tests with more complex expressions, e.g., `(!(1 + 2 > 3)) ? (2 > 1 : 1 + 1 : 0) : (2 > 1 : -1 + -1 : 0)` should be `2`.